src/OpenTelemetry.Instrumentation.AWS/Implementation/Metrics/AWSHistogram.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using Amazon.Runtime.Telemetry;
+using Amazon.Runtime.Telemetry.Metrics;
+
+namespace OpenTelemetry.Instrumentation.AWS.Implementation.Metrics;
+
+internal class AWSHistogram<T> : Histogram<T>
```

Comment:
what is the need of these wrappers over existing Histogram and others? Metrics API are designed to support high performance, allocation-free scenarios, and this wrappers is negating some of the perf optimizations... Wondering if this is really needed?

Reply:
We needed these wrappers to allow the AWS SDK to call these methods directly, and unfortunately the AWS SDK cannot take on new dependencies, and we supported .NET Standard and .NET Core 3.1, which lack `System.Diagnostics.DiagnosticSource` that is available in newer versions.